documents/implementation-guide.md

```diff
@@ -0,0 +1,43 @@
+
+## Usage
+
+- Add [jQuery](https://jquery.com/) to your project
+
+- Add these files to your project:
+
+```
+https://style.binary.com/binary.js
+https://style.binary.com/binary.css
+```
+
+- If you need Date and Time Pickers, Tabs or Accordion include "binary.more.js" and "binary.more.css" in your project, this will include a copy of jquery-ui with customized styles.
+
+```
+https://style.binary.com/binary.more.js
+https://style.binary.com/binary.more.css
+```
+
+### NPM Usage (Recommended)
```

Comment:
Can we move this section to the top? Or perhaps add another subtitle for the above section (something like Manual Import😸 ). Currently, It makes it look like the above steps are required to proceed with NPM Usage, which I think is not the case.